app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -33,7 +34,10 @@ def generate_pdf(_assessed_data)
 
     # Override this method to prevent the submission from getting the PDF and special issue
     def release_pdf?
-      true
+      flipper_symbol = "rrd_#{@disability_struct[:flipper_name].downcase}_release_pdf".to_sym
+      return true unless Flipper.exist?(flipper_symbol)
```

Comment:
By default, if Flipper symbol doesn't exist (e.g., `rrd_hypertension_release_pdf`), return true. We want this default so we can remove the Flipper flag once we're done experimenting with the disability.